src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,21 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Compares Person objects, according to Person names
+     * @param other Other Person object
+     * @return a negative integer, zero, or a positive integer as
+     * this Person is less than, equal to, or greater than the other Person
+     */
+    public int compareTo(Person other) {
+        String thisPersonName = this.getName().toString();
+        String otherPersonName = other.getName().toString();
+        if (thisPersonName.compareTo(otherPersonName) > 0)
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. brace style problem.